contrib/local-environment/docker-compose-keycloak.yaml

```diff
@@ -43,7 +43,7 @@ services:
 
   keycloak:
     container_name: keycloak
-    image: jboss/keycloak:10.0.0
+    image: jboss/keycloak:10.0.2
```

Comment:
Needs to be updated to 16.1.1 at some point. Probably best if someone familiar with keycloak could do this.

Reply:
Keycloak is already at version 23 😅

This needs to be updated!